### PR TITLE
Add PR submitter checklist to Pull Request template

### PR DESCRIPTION
**Pull request by cfarm**
Add our PR submitter checklist to the Pull Request template so we can easily check off the requirements of each PR when we submit it.

## Additions

- PR checklist for submitter
- added a link in our Code Review standards to bookmarklet code for checklists reviewers and for submitters in Gists (should we add these to the repo?)

## Removals

- removed placeholder text from the PR template, so it's easier to copy, paste, update.

## Changes

- changed "Notes" to "Todos" since that's how I've been using it. I put any generic "notes" in the intro paragraph. Todos is for upcoming related PRs, or tasks to finish in the PR.

## Testing

- save bookmarklet code to your browser bar, try to make a PR with it.

## Review

- @ascott1 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices 
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by cfarm** [comment]
I am bad at Github.com interface and accidentally committed a related change to master: https://github.com/cfpb/front-end/blob/master/code-reviews.md

**Comment by jimmynotjim** [comment]
@cfarm eh, not like this repo has deployable code, so no big deal. Mind adding some semantic line breaks to that in this PR though?

**Comment by ascott1** [comment]
This is great @cfarm! I'm gonna merge @jimmynotjim , because none of the new text is longer than 2 lines.

**Comment by jimmynotjim** [comment]
@ascott1 sorry, I was talking about the accidental changes in 17ed2de

**Comment by ascott1** [comment]
Ohhhhh. Cool. Yes plz @cfarm.

File: pr-template.md [hunk 1]
```diff
@@ -2,43 +2,40 @@ Short description explaining the high-level reason for the pull request
...
 
-## Preview
+## Screenshots
+
+
 
-Include any screenshots that will make reviewing/testing easier
+## Todos
 
-[Preview this PR without the whitespace changes](?w=0)
```
Comment (Scotchester) [inline pr-template.md]: Was this removal accidental? I'd like to reinstate it.

**Pull request merged**